pkg/services/librarypanels/librarypanels.go

```diff
@@ -52,7 +52,8 @@ func (lps *LibraryPanelService) AddMigration(mg *migrator.Migrator) {
 			{Name: "id", Type: migrator.DB_BigInt, IsPrimaryKey: true, IsAutoIncrement: true},
 			{Name: "org_id", Type: migrator.DB_BigInt, Nullable: false},
 			{Name: "folder_id", Type: migrator.DB_BigInt, Nullable: false},
-			{Name: "title", Type: migrator.DB_NVarchar, Length: 255, Nullable: false},
+			{Name: "uid", Type: migrator.DB_NVarchar, Length: 40, Nullable: false},
```

Comment:
Since we're updating the table structure, don't we need to have a new table version (or alternatively just some `ALTER TABLE` commands)? IE
```go
mg.AddMigration("Add uid column", NewAddColumnMigration(libraryPanelV1, &Column{
	Name: "library_panel", Type: DB_NVarchar, Length: 40, Nullable: false,
}))

mg.AddMigration("Rename title column", ...
```

For places where this migration has already successfully run, this would fail AFAICT

Reply:
That is correct, but as this is still behind a feature flag, we're taking this risk.

Reply:
@wbrowne I think the idea is we can ignore failing migrations with this PR, since this feature is only enabled by developers.

Reply:
Always read the label! My bad. Looking good in that case @hugohaggmark!